src/CalcViewModel/ApplicationViewModel.cpp

```diff
@@ -200,3 +202,57 @@ void ApplicationViewModel::SetMenuCategories()
     // property setter logic.
     Categories = NavCategoryGroup::CreateMenuOptions();
 }
+
+void ApplicationViewModel::AlwaysOnTopButtonClick(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
+{
+    HandleAlwaysOnTopButtonClick(sender, e);
+}
+
+Concurrency::task<void> ApplicationViewModel::HandleAlwaysOnTopButtonClick(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
+{
+    if (ApplicationView::GetForCurrentView()->GetForCurrentView()->ViewMode == ApplicationViewMode::CompactOverlay)
+    {
+        CalculatorViewModel->AreHistoryShortcutsEnabled = true;
```

Comment:
Could you do something similar to this property for memory shortcuts instead of trying to keep them disabled in AoT mode?